### PR TITLE
Feat/paneu updates

### PR DESCRIPTION
**Pull request by persas**
All the fixes we saw in the meeting

### CODE REVIEW

File: src/test/dataMigration/stepsPanEuropean/migrateForestServiceCategoryValues.ts [hunk 1]
```diff
@@ -15,23 +15,22 @@ export const migrateForestServiceCategoryValues = async (props: Props, client: B
...
-                            left join ${schemaAssessment}.col c on n.col_uuid = c.uuid
-                            left join ${schemaAssessment}.row r on r.id = c.row_id
-                            left join ${schemaAssessment}."table" t on r.table_id = t.id
-                   where c.props ->> 'colName' = 'forest_service_category'
-                     and n.value ->> 'raw' is not null),
-             b as (select a.id, a.value, a.rep[1], replace(initcap(replace(a.rep[1], '_', ' ')), ' ', '') as val
-                   from a)
-        update ${schemaCycle}.node n
-        set value = jsonb_set(n.value, '{raw}', to_jsonb(val.key))
+      with a as (select distinct n.id,
```
Comment (minotogna) [inline src/test/dataMigration/stepsPanEuropean/migrateForestServiceCategoryValues.ts]: why has this changed ? difficult to see diffs with different formatting

File: src/server/service/socket/socketService.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ const init = (server: http.Server): void => {
   //   console.log('==== NEW CONNECTION', socket)
   // })
 
-  io.engine.on('connection_error', (err) => {
+  io.engine.on('connection_error', (err: any) => {
```
Comment (minotogna) [inline src/server/service/socket/socketService.ts]: is this changes needed ? I'd rather not use any. but check if there's a type in case you want to add it.

File: src/test/dataMigration/stepsPanEuropean/migrateForestServiceCategoryValues.ts (hunk 1 above)
Comment (persas) [reply src/test/dataMigration/stepsPanEuropean/migrateForestServiceCategoryValues.ts]: It was failing, sorry for the formatting, was automatic. Basically, the only change was adding `as` here:

image

File: src/server/service/socket/socketService.ts (hunk 2 above)
Comment (persas) [reply src/server/service/socket/socketService.ts:23]: I rather not use any either. But type Error didn't have `.code` and it's used below, and it was failing the migration if nothing changes.

Comment (persas) [reply src/server/service/socket/socketService.ts]: I rather not use any either. But type Error didn't have .code and it's used below, and it was failing the migration if nothing changes.

File: src/test/dataMigration/stepsPanEuropean/migrateForestServiceCategoryValues.ts (hunk 1 above)
Comment (minotogna) [reply src/test/dataMigration/stepsPanEuropean/migrateForestServiceCategoryValues.ts]: what version of Postgres and node are you using ?

**Review by minotogna — APPROVED** [review]
